lib/stdlib/test/timer_SUITE.erl

```diff
@@ -168,7 +168,7 @@ s_a_t(C, TimeOut) ->
 a_t(C, TimeOut) ->
     start_watchdog(self(), TimeOut),
     Start = system_time(),
-    timer:send_after(TimeOut, self(), now),
+    {ok, {once, _}} = timer:send_after(TimeOut, self(), now),
```

Comment:
Testing undocumented implementation details (`{ok, {once, _}}` instead of `{ok, _}`) is OK and may find more bugs which is always good, but I think you should add a comment at the beginning of the module to mention that you do that and that the test suite might need to be updated if those implementation details are changed.

Reply:
Added in last commit.